HISTORY.rst

```diff
@@ -1,6 +1,14 @@
 History
 =======
 
+Unreleased
```

Comment:
I usually try to plan out the next release. In this case it's starting out as a bug fix so I'd do:

```
0.8.1 - 2017-xx-yy
------------------
```

This also means less fiddling with the header to make it valid RST and such.